discordjs/discordjs-typescript/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "axoncore-app",
+  "version": "1.0.0",
+  "description": "Boilerplate AxonCore app [discordjs-esm].",
+  "author": "KhaaZ",
+  "link": "https://github.com/Khaazz/AxonCore",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "dependencies": {
+    "axoncore": "git+https://www.github.com/khaazz/axoncore.git#dev",
```

Comment:
We will want to switch this to master as well.
And same for all other package.json iteration if not done already